blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxWriter.java

```diff
@@ -376,4 +378,13 @@ public void insertRollups(ArrayList<SingleRollupWriteContext> writeContexts) thr
             ctx.stop();
         }
     }
+    public void updateTTL(ColumnFamily CF, Locator l, ColumnList<Long> cols, Integer ttl)
```

Comment:
NIT: It will be better to have self-descriptive name of parameters, especially for the locator.